### PR TITLE
feedback for enter, landing, leaderboard, teams, faq, manifesto ship

### PR DESCRIPTION
**Pull request by towardsnewtech**

### CODE REVIEW

File: components/LeaderboardPage/LeaderboardRow.tsx [hunk 1]
```diff
@@ -20,18 +23,46 @@ const LeaderboardRow = ({ address, numberOwned, rank, twitterHandle }) => {
 
   return (
     <tr key={address} className="text-center bg-white hover:bg-blue-300">
-      <td className="px-4 py-2 border-r-2 border-black">#{rank}</td>
-      <td className="px-4 py-2 border-r-2 border-black">{numberOwned}</td>
-      <td className="flex items-center gap-3 px-4 py-2 border-r-2 border-black">
-        <PFP address={address} height={25} width={25} />
+      <td
+        className="
+        text-[8px] xs:text-[11px] md:text-[16px]
```
Comment (sweetmantech) [inline components/LeaderboardPage/LeaderboardRow.tsx]:
```suggestion
        className="text-[8px] xs:text-[11px] md:text-[16px]
```

**Review by SweetmanTech — APPROVED** [review]
ship it 🚢

File: components/LeaderboardPage/LeaderboardRow.tsx [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+import { useMediaQuery } from "usehooks-ts"
 import retryGetEns from "../../lib/retryGetEns"
 import truncateEthAddress from "../../lib/truncateEthAddress"
 import PFP from "../PFP"
 
 const LeaderboardRow = ({ address, numberOwned, rank, twitterHandle }) => {
   const [ensName, setEnsName] = useState(null as string)
 
+  const isMobile = useMediaQuery("(max-width: 768px)")
+  const isIphone = useMediaQuery("")
```
Comment (crazzywizard) [inline components/LeaderboardPage/LeaderboardRow.tsx]: what is the purpose of `isIphone`?

File: components/LeaderboardPage/LeaderboardRow.tsx (hunk 1 above)
Comment (towardsnewtech) [reply components/LeaderboardPage/LeaderboardRow.tsx]: fixed that part.

**Review by crazzywizard — APPROVED** [review]

**Pull request merged**